### ISSUE TITLE
Improve sample code for C++ Hello World

### ISSUE THREAD

**Issue by michelleangela** [microsoft/vscode-docs#4771]
using `using namespace std;` and outputting a new line with `std::endl` is bad practice in modern C++ and should not be shown to beginners who will think that is the only way to output a new line. it would be better to use

```
#include <iostream>

int main(){
    std::cout << "Hello, world!\n";
}
```

as a compile sanity check instead.

https://code.visualstudio.com/docs/languages/cpp#_add-hello-world-source-code

_Originally posted by @PeterSommerlad in https://github.com/microsoft/vscode-cpptools/discussions/8029_

**Comment by bobbrow** [comment]
I agree with abolishing `using namespace std`, but the line is a bit more fuzzy with `std::endl`.  For examples that cover debugging, having an `endl` will ensure that lines are flushed to stdout as you step over them.  Switching to `\n` can cause confusion for beginners.

**Comment by gregvanl** [comment]
Just a note that changing the code block will require several new screenshots.
